### PR TITLE
[BT-652]: Improve Home page and Navigation.

### PR DESCRIPTION
**Pull request by cjcenizal**
image

image

### CODE REVIEW

File: src/guide/components/navigation/Navigation.jsx [hunk 1]
```diff
@@ -15,80 +15,73 @@ export default class Navigation extends Component {
...
-          </NavButton>
+          </NavItem>
         );
       }
-    }.bind(this);
+    };
 
-    const navItems = [];
+    const componentNavItems = [];
+    addRoutesToNav('Components', this.props.componentRoutes, componentNavItems);
```
Comment (jenslonkowski) [inline src/guide/components/navigation/Navigation.jsx]: Please change this to multi-line to match lines 43-48.

**Comment by jenslonkowski** [comment]
Great improvements! 👍

**Comment by ahollesen** [comment]
Yup, looks nice 👍

File: src/guide/components/navigation/Navigation.jsx (hunk 1 above)
Comment (cjcenizal) [inline src/guide/components/navigation/Navigation.jsx]: Sure, but lines 43-48 were only changed to adhere to the linter.

Comment (jenslonkowski) [inline src/guide/components/navigation/Navigation.jsx]: I see. But I think in general it's a good idea to use multiple lines if there are multiple arguments. Would you agree?

Comment (cjcenizal) [inline src/guide/components/navigation/Navigation.jsx]: It depends on the situation for me. I think formatting this to match 43-48 is a good idea.

**Comment by cjcenizal** [comment]
Thanks guys!

**Comment by plain** [comment]
CJ - the design looks great. I may want to tweak it in the future, but I am fine with this to get it out the door.

**Comment by cjcenizal** [comment]
@plain Awesome! Thanks for taking a look. I'm sure you can really pimp it out... I didn't spend a lot of time on the look and feel.

**Pull request merged**